ViennaAdvantage/Areas/VA012/Models/BankStatementModel.cs

```diff
@@ -677,11 +642,15 @@ FROM C_PERIOD
                         {
                             if (_existingAccountID > 0 && _existingAccountID != _formData[0]._cmbBankAccount)
                             {
+                                //closing transaction
+                                trx.Rollback();
                                 return "VA012_StatementAlreadyExistDiffAcc";
                             }
                         }
                         else if (_statementDocStatus == "CO" || _statementDocStatus == "CL" || _statementDocStatus == "RE" || _statementDocStatus == "VO")
                         {
+                            //closing transaction
+                            trx.Rollback();
```

Comment:
before return, close the self created transaction

Reply:
Done.